core/storage/btree.rs

```diff
@@ -9715,4 +9831,357 @@ mod tests {
         )
         .unwrap();
     }
+
+    #[test]
+    pub fn test_clear_btree_empty_table() {
+        let (pager, root_page, _, _) = empty_btree();
+        let num_columns = 5;
+        let mut cursor = BTreeCursor::new_table(None, pager.clone(), root_page, num_columns);
+
+        // Clear empty table
+        let cell_count = run_until_done(|| cursor.clear_btree(), &pager).unwrap();
+        assert_eq!(cell_count, 0, "Empty table should have 0 cells cleared");
+
+        // Verify table is still empty by checking the cursor's empty state directly
+        assert!(
+            cursor.is_empty(),
+            "Table should still be empty after clearing"
+        );
+    }
+
+    #[test]
+    pub fn test_clear_btree_single_page_table() {
+        let (pager, root_page, _, _) = empty_btree();
+        let num_columns = 5;
+        let mut cursor = BTreeCursor::new_table(None, pager.clone(), root_page, num_columns);
+
+        // Insert 10 records
+        let num_records = 10;
+        for i in 1..=num_records {
+            let regs = &[Register::Value(Value::Integer(i))];
+            let record = ImmutableRecord::from_registers(regs, regs.len());
+            let key = BTreeKey::new_table_rowid(i, Some(&record));
+            run_until_done(|| cursor.insert(&key, false), &pager).unwrap();
+        }
+
+        // Verify records exist by checking a specific record
+        let exists = run_until_done(|| cursor.exists(&Value::Integer(1)), &pager).unwrap();
```

Comment:
no need for this i think, we should have other tests that verify that an inserted record is actually inserted

Reply:
I ended up removing it